library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -150,7 +151,14 @@ public void onCreate() {
 
         downloadClientReadyChecker = getDownloadClientReadyChecker();
 
-        mNotifier = new DownloadNotifier(this, getNotificationImageRetriever(), getResources());
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        NotificationDisplayer notificationDisplayer = new NotificationDisplayer(
+                this,
+                notificationManager,
+                getNotificationImageRetriever(),
+                getResources());
+
```

Comment:
the creation of the `notificationDisplayer` is linked to the `new DownloadNotifier` so why is there whitespace in the middle?